mod/05_email/set/type/email_template.rb

```diff
@@ -5,7 +6,7 @@ def clean_html?
 def deliver args={}
   begin
     mail = format.render_mail(args)
-    mail.deliver 
+    mail.deliver
```

Comment:
It's worth discussing whether it would be better to wrap the above in an Auth.as_bot block.  Issue is whether the mail view can render anything that we wouldn't want an unpermissioned user to see.

Currently tempted to make that change...